react_ws_src/src/helpers/__tests__/computer_moves-test.js

```diff
@@ -0,0 +1,114 @@
+jest.unmock('../computer_moves');
+import {
+  convert_cell_to_index,
+  convert_index_to_cell,
+  convert_move_coordinates_to_move_position_array,
+  get_available_moves,
+  get_board, get_computer_move,
+  get_minimax,
+  is_winner,
+  normalize_move_coordinates,
+} from '../computer_moves';
```

Comment:
Ideally I wouldn't have exported all the internal functions (in OO, they'd be considered `private`), but I think it was better to break things down and test them individually rather than trying to handle all the permutations via the main `get_computer_move` function.